### DISCUSSION TITLE
Add astral sorcery crystals to laser drill output

### DISCUSSION THREAD

**Discussion by FilJ00** [InnovativeOnlineIndustries/Industrial-Foregoing#1031] [Feature Requests]
Just a QOL change for making modpacks.

**Comment by MuteTiefling** [comment]
You know you can add these yourself, right?

**Reply by FilJ00** [reply]
Oh
No, I don't. 
Couldn't find anywhere to do so tho.

**Reply by MuteTiefling** [reply]
If you're making a pack, you need to learn how to work with data packs. That's how everything works these days. With data packs your can add, remove, or tweak existing recipes.

Not going to give you a tutorial here. Look into kubejs or crafttweaker to learn how.
